### PR TITLE
refactor: Update categories color

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

**Comment by kosssi** [comment]
Ne pas merger il y a un bug dans les imports de cozy-ui

File: src/ducks/categories/categoriesMap.js [hunk 1]
```diff
@@ -6,45 +6,51 @@
 
 import tree from './tree'
 
+const getCssVariableValue = variableName =>
```
Comment (drazik) [inline src/ducks/categories/categoriesMap.js]: Shouldn't it be placed in a more global place? I think we will reuse it elsewhere pretty quickly.

Comment (kosssi) [reply src/ducks/categories/categoriesMap.js]: https://github.com/cozy/cozy-ui/pull/763/files

**Comment by kosssi** [comment]
@drazik yes I check with @joel-costa ;)

File: src/ducks/categories/categoriesMap.js [hunk 2]
```diff
@@ -5,46 +5,47 @@
 */
 
 import tree from './tree'
+import { getCssVariableValue } from 'cozy-ui/react/utils/color'
```
Comment (kosssi) [inline src/ducks/categories/categoriesMap.js]: This function is on this PR https://github.com/cozy/cozy-ui/pull/763

File: src/ducks/categories/categoriesMap.js [hunk 3]
```diff
@@ -5,46 +5,47 @@
...
-    color: '#FF962F'
+    color: getCssVariableValue('mango')
   },
   incomeCat: {
-    color: '#35CE68'
+    color: getCssVariableValue('emerald')
   },
   activities: {
-    color: '#FC4C83'
+    color: getCssVariableValue('fuchsia')
```
Comment (kosssi) [inline src/ducks/categories/categoriesMap.js]: This variable is on this PR https://github.com/cozy/cozy-ui/pull/760

**Comment by kosssi** [comment]
We can not merge this PR since we can not update properly cozy-ui https://github.com/cozy/cozy-ui/issues/762

**Pull request merged**